lib/hammer_cli_foreman_remote_execution/job_invocation.rb

```diff
@@ -20,6 +20,7 @@ def self.included(base)
           field :total, _('Total')
           field :start_at, _('Start')
           field :randomized_ordering, _('Randomized ordering')
+        field :inputs, _('Inputs')
```

Comment:
Also, if you don't want to show this field all the time, but only if data are available, then try to set `hide_blank` to true.